client/src/driver/BitbakeESDK.ts

```diff
@@ -85,3 +86,27 @@ export function generateTasksDefinitions (workspace: DevtoolWorkspaceInfo, bitba
   logger.info(`Generated ${vscodeTasksPath} for ${recipeName}`)
   logger.debug(`Tasks: ${JSON.stringify(vscodeTasks)}`)
 }
+
+export async function generateCPPProperties (workspace: DevtoolWorkspaceInfo, bitBakeProjectScanner: BitBakeProjectScanner, client: LanguageClient): Promise<void> {
+  const vscodeCppPropertiesPath = path.join(workspace.path, '.vscode', 'c_cpp_properties.json')
+  const vscodeCppProperties = loadJsonFile(vscodeCppPropertiesPath)
```

Comment:
this is of type `any`, which means it could be undefined, then the `vscodeCppProperties.configurations` at `L105` could throw an type error

Reply:
It's ok to throw in a command, it will produce an error notification for the user which is what we want. The question is, is that message useful?
To test this, I put invalid JSON content in the file. It produces this error which I think is explicit
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/8cd2380c-f406-43c8-9e69-ce613ae4e373)
It just misses the file name. I added a commit to also print the name.

Reply:
oh, nice. I am good with it.